opsdroid/helper.py

```diff
@@ -19,7 +23,12 @@ def get_opsdroid():
 
 
 def del_rw(action, name, exc):
-    """Error handler for removing read only files."""
+    """This method return an error handler for removing.
+        This method allow users to read only files.
```

Comment:
This isn't the case, this function basically removes the readonly flag of a file (called name)

In here you should add the following text in the docstrings:


```
Args:
    action: 
    name:
    exc:
```

Also, I would mention that this function is mostly used on tests (this can help people when they are looking at code and wonder what the hell this is used for).